data_analysis/README.md

```diff
@@ -1,66 +1,41 @@
-        * `route_id`: Route identifier, from GTFS schedule 
-        * `day_type`: A short code identifying day type: holiday (`hol`), weekday (`wk `), Saturday (`sat`), or Sunday (`sun`)
-        * `trip_count_rt`: Total realtime trips identified for this route + day type -- TODO: currently this total is calculated by summing the hourly totals; this could be improved (but the method is the same for scheduled and realtime, so it is apples-to-apples)
-        * `trip_count_sched`: Total scheduled trips for this route + day type -- TODO: currently this total is calculated by summing the hourly totals; this could be improved (but the method is the same for scheduled and realtime, so it is apples-to-apples)
-        * `ratio`: `trip_count_rt` / `trip_count_sched`
-    * In S3, these are available in the `chn-ghost-buses-public` bucket in a folder called `schedule_rt_comparisons`. Within that folder there is one file per published GTFS feed version from `2022-05-07` to `2022-07-18`. A list of feed versions is available on [Transit.Land](https://www.transit.land/feeds/f-dp3-cta). The full S3 filepaths are like: `schedule_rt_comparisons/route_level/schedule_v20220507_realtime_rt_level_comparison_2022-05-20_to_2022-06-02.csv`. An overall combined file is available at: `schedule_rt_comparisons/route_level/combined_schedule_realtime_rt_level_comparison_2022-05-20_to_2022-07-20.csv`.
\ No newline at end of file
+        * These files are generated daily between 10 and 11am Central for the prior. There is one file per full day from `2022-05-20` until the day before you are making the request. So, if you are checking on `2022-10-02` after 11am Central, data will be available up to and including `2022-10-01`.
+        * In S3, these are available in the `chn-ghost-buses-public` bucket in a folder called `bus_full_day_data_v2`. Full filenames are like `bus_full_day_data_v2/{date in YYYY-MM-DD format}.csv`. So, to load the data for `2022-10-01` in Pandas, you could do: `pandas.read_csv(https://chn-ghost-buses-public.s3.us-east-2.amazonaws.com/bus_full_day_data_v2/2022-10-01.csv)`.
+    * **Errors**: 
+        * These are CSV files that contain all the error data we received from the API, concatenated together for a full day. The schema of the data is exactly what is returned from the API, with only a `scrape_file` field (see above) added that records the name of the S3 resource where the original JSON response that contained this row is saved; see [the `getvehicles` section of the CTA documentation](https://www.transitchicago.com/assets/1/6/cta_Bus_Tracker_API_Developer_Guide_and_Documentation_20160929.pdf) for field definitions from the API.
+        * These files are generated daily between 10 and 11am Central for the prior. There is one file per full day from `2022-05-20` until the day before you are making the request. So, if you are checking on `2022-10-02` after 11am Central, errors will be available up to and including `2022-10-01`.
```

Comment:
Add the word "day" after "prior" to "These files are generated daily between 10 and 11am Central for the prior."

Reply:
Ah good catch, fixed